pkg/userdata/ubuntu/testdata/docker-1.13-dist-upgrade-on-boot-aws.golden

```diff
@@ -124,159 +197,7 @@ write_files:
     [Service]
     Type=oneshot
     RemainAfterExit=true
-    ExecStart=/usr/local/bin/download-kubelet-kubeadm-unitfile
-
-- path: "/etc/systemd/system/kubeadm-join.service"
-  content: |
-    [Unit]
-    Requires=network-online.target
-    Requires=kubelet-binary.service kubeadm-binary.service cni-binary.service kubeadm-unitfile.service
-    After=network-online.target
-    After=kubelet-binary.service kubeadm-binary.service cni-binary.service kubeadm-unitfile.service
-
-
-    [Service]
-    Type=oneshot
-    RemainAfterExit=true
-    Environment="PATH=/sbin:/bin:/usr/sbin:/usr/bin:/opt/bin"
-    ExecStartPre=/sbin/modprobe br_netfilter
-    ExecStartPre=/sbin/sysctl --system
-    ExecStart=/opt/bin/kubeadm join \
-      --token my-token \
-      --discovery-token-ca-cert-hash sha256:6caecce9fedcb55d4953d61a27dc6997361a2f226ad86d7e6004dde7526fc4b1 \
-      server:443
-
-- path: "/etc/systemd/system/kubelet.service.d/20-extra.conf"
-  content: |
-    [Service]
-    Environment="KUBELET_EXTRA_ARGS=--cloud-provider=aws --cloud-config=/etc/kubernetes/cloud-config \
-      --authentication-token-webhook=true --hostname-override=node1 --read-only-port 0 \
-      "
-
-- path: "/etc/systemd/system/kubelet.service.d/30-clusterdns.conf"
-  content: |
-    [Service]
-    Environment="KUBELET_DNS_ARGS=--cluster-dns=10.10.10.10 --cluster-domain=cluster.local"
-
-- path: "/etc/systemd/system/kubelet.service"
-  content: |
-    [Unit]
-    Description=Kubelet
-    Requires=network-online.target
-    After=network-online.target
-    Requires=kubelet-binary.service kubeadm-binary.service cni-binary.service kubeadm-unitfile.service
-    After=kubelet-binary.service kubeadm-binary.service cni-binary.service kubeadm-unitfile.service
-    Requires=docker.service
-    After=docker.service
-
-    [Service]
-    Environment="PATH=/sbin:/bin:/usr/sbin:/usr/bin:/opt/bin"
-    ExecStart=/opt/bin/kubelet
-    Restart=always
-    StartLimitInterval=0
-    RestartSec=10
-    Restart=always
-
-    [Install]
-    WantedBy=multi-user.target
+    ExecStart=/usr/local/bin/supervise.sh /usr/local/bin/setup
 
 runcmd:
```

Comment:
Please add a install section to the install service and use systemctl enable - - now l install.service here, otherwise this won't get started after a reboot e. G. due to updates. Other than that, really cool approach!

Reply:
Also please call systemctl enable for both kubelet and docker in the setup script